evap/locale/de/formats.py

```diff
@@ -1 +1 @@
-from evap.locale.en.formats import *  # pylint: disable=unused-wildcard-import,wildcard-import
+from evap.locale.en.formats import *  # pylint: disable=unused-wildcard-import,wildcard-import  # noqa: F403
```

Comment:
I'd tend to not put (and thus maintain) multiple ignores for "the same thing" in the codebase. Initially, my idea was to completely remove pylint (with the intention of getting rid of the long runtime).

If ruff is still lacking features, we can run both, but then I'd try to configure them in such a way that pylint doesn't check for anything that ruff also checks for. (I don't know how much configuration that requires though. I'm not interested in maintaining hundreds of lines of linter configuration either).

Reply:
in [3de4519](https://github.com/e-valuation/EvaP/pull/2051/commits/3de451916f54fda5139a303efe7fc3d099244777), I disabled three pylint checks in favor of ruff. There should be no cases where of both ruff and pylint is disabled